src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -1165,18 +1179,34 @@ public void testCreatePaymentStatusTransition() throws Exception {
         HyperwalletStatusTransition returnValue;
         try {
             returnValue = client.createPaymentStatusTransition("pmt-2c059341-8281-4d30-a65d-a49d8e2a9b0f",
-                                                                transition);
+                    transition);
         } catch (Exception e) {
             mockServer.verify(parseRequest(functionality));
             throw e;
         }
+        List<HyperwalletLink> hyperwalletLinks = new ArrayList<>();
+        HyperwalletLink hyperwalletLink = new HyperwalletLink();
+        hyperwalletLink.setHref(
+                "https://api.sandbox.hyperwallet.com/rest/v4/payments/pmt-8716ee36-8b8f-4572-bf68-b9d1f956a3a2/status-transitions/sts-46592b6c-7d8b"
+                        + "-4e00-abf6-ca5a03880d31");
+        Map<String, String> mapParams = new HashMap<>();
+        mapParams.put("rel", "self");
+        hyperwalletLink.setParams(mapParams);
+        hyperwalletLinks.add(hyperwalletLink);
 
         assertThat(returnValue.getToken(), is(equalTo("sts-46592b6c-7d8b-4e00-abf6-ca5a03880d31")));
         assertThat(returnValue.getCreatedOn(), is(equalTo(dateFormat.parse("2017-12-21T11:35:43 UTC"))));
         assertThat(returnValue.getTransition(), is(equalTo(CANCELLED)));
         assertThat(returnValue.getFromStatus(), is(equalTo(SCHEDULED)));
         assertThat(returnValue.getToStatus(), is(equalTo(CANCELLED)));
         assertThat(returnValue.getNotes(), is(equalTo("Cancel a payment upon customer request.")));
+        if (returnValue.getLinks() != null) {
+            HyperwalletLink actualHyperwalletLink = returnValue.getLinks().get(0);
+            HyperwalletLink expectedHyperwalletLink = returnValue.getLinks().get(0);
+            assertThat(actualHyperwalletLink.getHref(), is(equalTo(expectedHyperwalletLink.getHref())));
```

Comment:
This statement and the following two statements always return true as you are setting the value from returnValue.getLinks().get(0) to actualHyperwalletLink and expectedHyperwalletLink. Pls check the returnValue's link to that of hyperWalletLink

Reply:
Done.